.ai-bot/ui-workflows.md

```
@@ -0,0 +1,293 @@
+  └─ [static-ip-host-configurations]
+  └─ [static-ip-yaml-view]
+operators
+host-discovery
+storage
+networking
+[custom-manifests]   ← only if user adds manifests
+review
+```
+
+### Day 2 wizard steps (modal, `Day2Wizard.tsx`)
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)